blocks.md

```diff
@@ -0,0 +1,80 @@
+
+<h3>Developer Expectations</h3>
+
+* Developers will have a concrete set of requirements and guidelines to follow when writing blocks for the Block Directory
+
+<h3>Definitions</h3>
+
+First up, some clarity. For the purposes of the Block Directory, we distinguish between two types of plugins:
+1. Plugins that exist solely to distribute a block.
+1. All other plugins, including those that bundle many independent blocks, plugins that contain blocks in addition to other functionality, and plugins with no blocks at all.
+
+These guidelines apply specifically to the first type of plugin, which we’ll call here a Block Plugin. If your plugin is of the second type, then the further guidelines and restrictions do not apply to your work. All plugins, be they block-only or not, are required to comply with the Detailed Plugin Guidelines https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/.
+
+<h3>Block Plugins and the Block Directory</h3>
+
+The Block Directory contains only Block Plugins, that is to say plugins that contain only a single independent block and a minimum of supporting code. Block plugins have the following characteristics:
+
+1. A specific type of WordPress plugin, with the same structure including a readme.txt file.
+1. Containing only blocks (typically one).
+1. Contain a minimum of server-side code. 
+1. Do not have any UI outside of the post editor.
+
+In addition to following the guidelines that apply to all WordPress plugins, Block Plugins that are submitted to the Block Directory must adhere to these guidelines:
+
+<h3>Guidelines</h3>
+
+<h4>1. Block Plugins are for the Block Editor.</h4>
+
+Your block plugins should contain a block, and a minimum of other supporting code. It should not contain any UX outside of the editor, such as WordPress options or `wp-admin` menus. Server-side code should be kept to a minimum.
+
+<h4>2. Block Plugins are separate blocks.</h4>
+
+Block plugins are intended to be single purpose, separate, independent blocks, not bundles or compendiums of many blocks. In most cases, a Block Plugin should contain only one (single) block. It should not contain multiple blocks that could reasonably be split up into separate, independent plugins.
+
+Block plugins may contain more than one block where a clearly necessary parent/child dependency exists; for example a list block that contains list item blocks.
+
+<h4>3. Plugin titles should reflect the block title.</h4>
+
+We want users to interact with blocks, not product names. Block Plugins should be named to reflect the block; for example "Image Grid" block, "Improved Slider", "Business Hours" block. Please avoid plugin names unrelated to the block itself like "Widgets Incorporated Block", and refrain from block names that include trademarks or names that you do not legally represent; for example you cannot name a block "Facerange Block" unless you work for Facerange and have their permission.
```

Comment:
If blocks should be generically named “Image Grid” how would a user differentiate one image grid block from another in the block listing? The differentiation provided by a product name is helpful especially when the functionality provided by the block might not be as straightforward/generic as a slider.

Reply:
Good point. But what about block collections? There are an increasing number of block collections and it is really difficult to find blocks if you can only see numerous "ultimate" block collections.

Reply:
Once this new selection screen gets off the ground, we may see the collections stall out a little bit and maybe become rarer over time in favor of newer individual blocks. For the moment, it's just the optimal way to be able to get people to download / buy blocks from a developer.

Reply:
> If blocks should be generically named “Image Grid” how would a user differentiate one image grid block from another in the block listing? The differentiation provided by a product name is helpful especially when the functionality provided by the block might not be as straightforward or generic as (say) a slider.

Agreed, a poorly executed block preview would make this problematic. On the other side, this could be addressed with the combination of the plugin's icon, author and credibility. I think if we put more emphasis on the non title bits that legitimize the plugin, we make it much more straightforward in finding the block you are specifically looking for or discovering the right block to build with.

Reply:
Rapid fire idea: Is there any value in having an `integrations` category in the block.json?

Reply:
I agree that block collections have no place in the block directory. 

However, if I have for example a charterboat calendar, charterboat listing, and charterboat grid item it makes sense for users to have a way of accessing all of these together. While they do not "require" each other, they are designed to work together and have limited functionality without all 3....

Are we really just relying on the alphabet to bring those up together if a user types in Charterboat? Seems less sophisticated than we could be.